### DISCUSSION TITLE
How to get Prefix Commands?

### DISCUSSION THREAD

**Discussion by AAYU6H** [discordjs/discord.js#6762] [Q&A] [answered]
### Feature

I wanted to know what is the syntax for creating prefix commands in Discord.js. Kindly Help

### Ideal solution or implementation

Kindly help with Prefix Commands in Discord.Js

**Comment by kyranet** [comment]
Check the guides for how to create slash commands [here](https://discordjs.guide/interactions/registering-slash-commands.html).

**Comment by almostSouji** [comment, marked as answer]
If by "prefix command" you mean messages relying on and parsing the message content, instead of slash commands, you can find old guides on the archived guide version: https://v12.discordjs.guide/command-handling/. Do note, however, that Discord is planning to hide message content behind a privileged intent for large bots (100+ servers) and that they will not grant the intent just so you can have "old" commands.

**Comment by ghost** [comment]
client.on('message', message => {
	if (!message.content.startsWith(prefix) || message.author.bot) return;

	const args = message.content.slice(prefix.length).trim().split(/ +/);
	const command = args.shift().toLowerCase();

	if (command === 'ping') {
		message.channel.send('Pong.');
	} else if (command === 'beep') {
		message.channel.send('Boop.');
	}
	// ...
});

this is the code you could use for small projects
